src/components/Badge/index.js

```diff
@@ -4,7 +4,7 @@ import StyledContainer from './styled/container';
 import Content from './content';
 
 /**
- * Badges are labels which hold small amounts of information.
+ * Badges are labels that hold small amounts of information. They have rounded corners.
```

Comment:
" They have rounded corners." I think this phrase should not be in the description.
I think that the description of the component must not be linked to the appearance.